### ISSUE TITLE
Launch Section 215 petition before April 15

### ISSUE THREAD

**Issue by OpenMatt** [MozillaFoundation/plan#426]
## Goal
*   **Launch a petition campaign on or before April 15**. Calling on Congress to stop mass collecting data on Americans.
*   **Success metrics / KPI**: 100,000 signatures 
*   **More context (optional)**: [http://mzl.la/launch_plan](http://mzl.la/launch_plan) and the overall campaign: [http://mzl.la/surveillance_brief](http://mzl.la/surveillance_brief)

## Roles
* Owner: @OpenMatt 
* Decision-maker: @valianttry
* Quality Lead: @davidascher 
* Design Lead: @iamjessklein 
* Development Lead: @ScottDowne 
* Informed: @DaveSteer @HPaulJohnson @KevZawacki 

# Deliverables
- [ ]  **1) Online petition page**  
   - [ ] build ticket: https://github.com/MozillaFoundation/movement-building/issues/25 (Scott)
   - [ ] security review ticket: https://github.com/MozillaFoundation/movement-building/issues/27 (David / Simon)
- [ ] **2) Update the advocacy.mozilla.org front page** (Jess)
   - [ ] ticket: https://github.com/MozillaFoundation/movement-building/issues/26

**Additional stretch deliverables:**
- [x] **3) Design snippet** (Jess)
   - [ ] ticket: https://bugzilla.mozilla.org/show_bug.cgi?id=1152133
- [ ] **4) Design tile** (Jess)
   - [ ] ticket: https://github.com/MozillaFoundation/movement-building/issues/33

**Comment by OpenMatt** [comment]
@simonwex: this is a late-breaking P1. We just learned of the April 15 deadline yesterday. Filing now so we can hopefully get a dev assigned quickly.

**Comment by OpenMatt** [comment]
Still need a development lead for this. @simonwex @davidascher can you advise?

**Comment by simonwex** [comment]
@OpenMatt I was PTO Thursday, Friday. I'll get this assigned and moving asap today.

**Comment by OpenMatt** [comment]
sweet. thanks wex

**Comment by simonwex** [comment]
@ScottDowne is going to tackle this one. If it looks like he needs backup, we'll rope @alicoding in. @OpenMatt could you schedule a kickoff?

**Comment by OpenMatt** [comment]
Awesome! I'll schedule a kick-off w. me and Scott.

**Comment by OpenMatt** [comment]
Scott and I did a quick kick-off in IRC -- we're good. He's making good progress. As is Jess on the other deliverable.

**Comment by valianttry** [comment]
Just checked with @iamjessklein about possibly re-purposing her eyeball design for this campaign. I am double checking with jean Collings over on moco side about whether they can help with design for additional deliverables:

1. Tile
2. Snippet icon

Here's stand-in snippet copy in case @iamjessklein wants to mock up a snippet with the eyeball:
* U.S. Congress votes soon on whether to end mass surveillance. Keeping us safe shouldn’t cost us our privacy. Take action now.

**Comment by valianttry** [comment]
One more thing, the confirmation page should not include the call Congress promotion. I marked it up here:
![sendto_mozilla_org___2015_fccvote_net_neutrality_thanks](https://cloud.githubusercontent.com/assets/9830371/7015059/84c7500a-dc82-11e4-89cb-feccf1f6acd9.png)

**Comment by OpenMatt** [comment]
thanks @valianttry : -- let us know if a MoCo designer is available for those two extra deliverables. We're short on design resources so pushing hard for MoCo to take those on would help.

**Comment by iamjessklein** [comment]
Here's a quick take at an eyeball snippet gif (in here so you can see the animation).

![output_nuipbe](https://cloud.githubusercontent.com/assets/535012/7025299/b72ce328-dd10-11e4-92aa-c665847a232f.gif)

[Here is an updated design](https://redpen.io/myb3ad380c9196493d) for the Advocacy page incorporating the famous "eyeball".

**Comment by OpenMatt** [comment]
wow -- fast work Jess. @valianttry : does this give you what you need?

**Comment by OpenMatt** [comment]
@valianttry : if you do end up requiring design work for Tiles as well, can you include the specs for that?

**Comment by valianttry** [comment]
@iamjessklein This is delightful! I've shared it with the moco team and will get back to you. On Tiles, I'm getting the specs.

**Comment by valianttry** [comment]
Tiles specs are here: https://wiki.mozilla.org/images/7/73/Partner_Specs_Directory_Tiles.pdf
I'm in a meeting, but perhaps @KevZawacki can help develop copy for it? There's a static state and then on-rollover state.

**Comment by OpenMatt** [comment]
@iamjessklein can you create a design using those specs?
@KevZawacki can you provide copy?

**Comment by OpenMatt** [comment]
Security review ticket for petition page filed here: https://github.com/MozillaFoundation/movement-building/issues/27

**Comment by KevZawacki** [comment]
@OpenMatt @valianttry 

Yes, definitely! Initial ideas:

(static / rollover)

+ Stand up to mass surveillance / Petition Congress to end Section 215
+ Tell Congress it's time to stop mass surveillance / Sign the petition to end Section 215
+ You can help stop mass surveillance / Tell Congress not to reauthorize Section 215
+ Protect your right to privacy / Take action now to stop Section 215
+ Congress votes soon on mass surveillance /  Stand up now and sign the petition
+ Sign the petition to end mass surveillance / Tell Congress how you feel
+ Stand up for your privacy / Sign the petition to end mass surveillance

**Comment by OpenMatt** [comment]
Sounds great. @valianttry can you pick one, and then Jess can design it up.

**Comment by iamjessklein** [comment]
I like the first 3 options because they relate nicely to the content on the Advocacy site, but yes, @valianttry please let me know which direction you'd like for me to mock up.

**Comment by valianttry** [comment]
I don't think we should say "section 215" in any of our shorter calls to action, it's not recognizable / jargon. We should just say "help end mass surveillance". I also don't like "Tell congress how you feel" -- it sounds wishy-washy and not impactful. I like these:

Stand up for your privacy / Sign the petition to end mass surveillance 
Tell Congress it's time to stop mass surveillance / Sign the petition
You can help stop mass surveillance / Sign the petition

**Comment by valianttry** [comment]
@iamjessklein our moco friends have launched a bug in some program called "Bugzilla" :smirk: here:
https://bugzilla.mozilla.org/show_bug.cgi?id=1152133

I think we should finalize our designs and copy here, and put the final assets into that bug. Do you think you could make 1-2 variations of the eyeball? Maybe a blue version? We can also re-purpose the advocacy bull horn.

@KevZawacki snippet copy is 180 characters maximum. Want to take a swing at a few options?

**Comment by iamjessklein** [comment]
@valianttry - just to confirm, I need to deliver on 3 things:

- [ ] the promo content on Advocacy 
- [ ] the snippet 
- [ ] the tile 

Is that correct?
cc/ @OpenMatt

**Comment by KevZawacki** [comment]
Certainly. Some snippet possibilities: 

+ Privacy is a right, not a privilege. Tell Congress it's time to end mass surveillance. Take action and sign the petition. 
+ Mass surveillance threatens our privacy and security. Tell Congress it's time to end bulk collection of  personal data. Sign the petition now. 
+ Stand up to the Patriot Act and help put an end to mass surveillance. Sign the petition to Congress.

**Comment by OpenMatt** [comment]
Let's clean up this plan ticket -- it's getting unwieldy. Let's be clear about final steps to close -- and get this done fast. 
* I've updated the first comment above -- everything we need to close this is there. 
* @iamjessklein : I'm moving all snippet-related work out of this ticket into the bugzilla ticket. Let's get that ticket closed asap. https://bugzilla.mozilla.org/show_bug.cgi?id=1152133
* @valianttry please be clear about what you need from Jess to close that snippet ticket. How many design variations do you really need? Please be clear -- let's close these tickets asap. One last round of revisions and done. 
* I'm filing a new ticket for the Tile work. And have moved everything tile-related out of this ticket  there. 
* @davidascher @simonwex please tap the mic on the security review ticket. I don't know what the next steps are there.

**Comment by OpenMatt** [comment]
We can get at least 3 out of these 4 deliverables closed today. Let's push so that this doesn't bleed into the next Heartbeat unnecesarily.

**Comment by OpenMatt** [comment]
* petition page: is ready. but blocked on MoCo privacy review. (where's that ticket?)
* snippet. is done. blocked on finding the right person at MoCo to review and approved. We don't know who that is.

**Comment by OpenMatt** [comment]
Petition page is blocked on MoCo Privacy review: https://bugzilla.mozilla.org/show_bug.cgi?id=1151659

**Comment by davidascher** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1151659 is the privacy review.
I think it's almost closed.

**Comment by OpenMatt** [comment]
Bug for updating moz.org front page: https://bugzilla.mozilla.org/show_bug.cgi?id=1152159

**Comment by davidascher** [comment]
privacy review bug closed

**Comment by hannahkane** [comment]
Does this belong in the current milestone?

**Comment by ScottDowne** [comment]
I think maybe we can even close this?
